update_worlds_and_protos.py

```diff
@@ -0,0 +1,80 @@
+import os
```

Comment:
Is this file used anywhere?
Because it has exactly the same name as the one added in `scripts/packaging`.
I tried to check but I found just references to the one in `scripts/packaging`.

If it used, then there are some improvements to apply:
* header `#!/usr/bin/env python` and license are missing in this file
* should be moved in the `scripts` folder (or any other folder) and not be located in the main folder

Reply:
I believe it's not used. I removed it.